recruiter/cli.py

```diff
@@ -1,11 +1,11 @@
 import json
 
 from argparse import ArgumentParser
-from candidate import Candidate
+from recruiter.candidate import Candidate
```

Comment:
@pvwoods I'm pretty new to Python. Does P3 not support these relative imports and or are fully qualified import paths the recommended standard?